### PR TITLE
Added Role ceph fuse for cloudvolume

### PR DESCRIPTION
**Pull request by supermeng**

### CODE REVIEW

File: playbooks/roles/ceph-fuse/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+- name: install ceph-fuse
+  yum: 
+    name: ceph-fuse-{{ceph_version}}
+    state: present
+
+- name: get kernal
```
Comment (xtao) [inline playbooks/roles/ceph-fuse/tasks/main.yaml]: `kernel` ?

File: playbooks/roles/ceph-fuse/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  command: uname -r
+  register: kernal_info
+
+- name: bakup /sbin/mount.fuse.ceph
```
Comment (xtao) [inline playbooks/roles/ceph-fuse/tasks/main.yaml]: `backup` ?

File: playbooks/roles/ceph-fuse/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+- name: install ceph-fuse
+  yum: 
+    name: ceph-fuse-{{ceph_version}}
+    state: present
+
+- name: get kernal
+  command: uname -r
+  register: kernal_info
```
Comment (xtao) [inline playbooks/roles/ceph-fuse/tasks/main.yaml]: `kernel_info` ?

Comment (xtao) [inline playbooks/roles/ceph-fuse/tasks/main.yaml]: `kernel_info` ?

**Comment by xtao** [comment]
`cloudvolume`  ?

**Comment by supermeng** [comment]
已修改，   英语太渣了 0.0

File: playbooks/roles/ceph-fuse/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    name: ceph-fuse-{{ceph_version}}
+    state: present
+
+- name: get kernel
+  command: uname -r
+  register: kernel_info
+
+- name: backup /sbin/mount.fuse.ceph
+  command: mv /sbin/mount.fuse.ceph /sbin/mount.fuse.ceph.bak
+  when: "'3.10.0' in kernal_info.stdout"
```
Comment (xtao) [inline playbooks/roles/ceph-fuse/tasks/main.yaml]: 还有 `kernal_info` ..

File: playbooks/roles/ceph-fuse/tasks/main.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+- name: update ceph mount file
+  shell: sed -e "s/cephargs='--'\`echo \$1 | sed 's\/,\/ --\/g'\`/fs_spec=\`echo \$1 | sed \'s\/:\/=\/g\'\` \ncephargs=\'--\'\`echo \$fs_spec | sed \'s\/,\/ --\/g\`/g" /sbin/mount.fuse.ceph.bak >/sbin/mount.fuse.ceph
+  when: "'3.10.0' in kernal_info.stdout"
```
Comment (xtao) [inline playbooks/roles/ceph-fuse/tasks/main.yaml]: re

File: playbooks/roles/ceph-fuse/defaults/main.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+ceph_version: 10.2.3
+
+ceph_account: lain
+ceph_password: password
+
+ceph_mountpoint: /lain
+client_mountpoint: /data/lain/cloudvolumes
```
Comment (panli889) [inline playbooks/roles/ceph-fuse/defaults/main.yaml]: 这个用 cloud-volumes 吧

**Pull request merged**